lib/kafo/answer_file.rb

```diff
@@ -0,0 +1,56 @@
+require 'yaml'
+
+module Kafo
+  class AnswerFile
+
+    attr_reader :answers, :filename, :version
+
+    def initialize(answer_filename, version: 1)
+      @filename = answer_filename
+      @version = version.nil? ? 1 : version
+
+      begin
+        @answers = YAML.load_file(@filename)
+      rescue Errno::ENOENT
+        KafoConfigure.exit(:no_answer_filename) do
+          puts "No answer file found at #{@filename}"
+        end
+      end
+
+      validate
```

Comment:
Adds some validation of the answer file up front and will play a bigger role in a Version 2 where the schema is more defined.

Reply:
I debated not calling this in the init and letting the user of this class call the validation.